### DISCUSSION TITLE
Theme

### DISCUSSION THREAD

**Discussion by CChurchill404** [OrchardCMS/OrchardCore#13030] [Q&A]
Sorry I'm fairly new to orchard, 

But is there a way to extend themes off of one base theme? 

Thank you :)

**Comment by hishamco** [comment]
Of course you can I remember that I extended the admin theme while ago, is there a particular thing you are looking for?

**Reply by CChurchill404** [reply]
Would an example be okay? :)

**Reply by hishamco** [reply]
Unfortunately it's not online, it was in my old hard disk which is corrupted last two weeks :(

**Reply by hishamco** [reply]
Simply you can specify the `BaseTheme` attribute in your manifest. I have few themes available in [Orchard Core Contrib](https://github.com/OrchardCoreContrib/OrchardCoreContrib.Themes) hope to create a new admin them in upcoming future

**Reply by CChurchill404** [reply]
So I was going through the docs based on this pointer, and I found 

The BaseTheme property means that when the custom admin theme is active Orchard Core Display Management will search both TheAdmin theme and MyAdminTheme for template alternates when displaying admin content.

Views in MyAdminTheme will override views in TheAdmin.

Does this mean it will only apply to the cms theming, or would it carry things from my Views directory over too? Sorry for the hassle :)

**Reply by hishamco** [reply]
> Views in MyAdminTheme will override views in TheAdmin.

Yes

**Reply by CChurchill404** [reply]
Ayy, I've done it. Thanks alot!
